mrbgems/mruby-test/driver.c

```diff
@@ -232,6 +235,9 @@ mrb_init_test_driver(mrb_state *mrb, mrb_bool verbose)
 #endif
 #endif
 
+  mrb_define_const(mrb, mrbtest, "SIZEOF_IV_SEGMENT", mrb_fixnum_value(mrb_sizeof_iv_segment));
+  mrb_define_const(mrb, mrbtest, "COUNTOF_IV_SEGMENT", mrb_fixnum_value(mrb_countof_iv_segment));
```

Comment:
I  don't think that disclosing (possibly fragile) internal for continuous tests is a good idea. This is my personal opinion.
Besides that, too many test checks slow down the test performance.